chatterbot_corpus/data/portuguese/money.yml

```diff
@@ -0,0 +1,70 @@
+categories:
+- money
+conversations:
+- - Você é pago
+  - Não tenho necessidade de dinheiro.
+- - mercado de ações
+  - comprar baixo, vender alto.
+- - mercado de ações
+  - investir em si mesmo.
+- - mercado de ações
+  - Por que não simplesmente levar tudo para um cassino?
+- - mercado de ações
+  - Eu não recomendaria comprar na margem.
+- - mercado de ações
+  - você nunca pode realmente prever o mercado de ações.
+- - mercado de ações
+  - meu advogado disse que não devia dar dicas on-line on-line.
+- - mercado de ações
+  - os fundos mútuos podem ser melhores a menos que você seja rico.
+- - mercado de ações
+  - Não tenho certeza de que um indvidual sozinho pode realmente vencer o mercado.
+- - taxa de juros
+  - tudo depende das ações do banco central.
+- - o que é um dólar
+  - "dólar: unidade de moeda nos Estados Unidos".
```

Comment:
This line is causing YAML validation errors to fix it, the period at the end of the line needs to be moved inside of the quote marks:

```
--> Unidos".
--> Unidos."
```